bsonnumpy/jsonsl/jsonsl.h

```diff
@@ -168,6 +168,7 @@ typedef enum {
     /** @private Private */
     JSONSL_SPECIALf_ZERO    = 1 << 11 | JSONSL_SPECIALf_UNSIGNED,
     /** @private */
+    JSONSL_SPECIALf_DASH    = 1 << 12,
```

Comment:
How come this wasn't part of the previous commit? I see this line here https://github.com/mongodb/mongo-c-driver/blob/1.17.0/src/libbson/src/jsonsl/jsonsl.h#L171?

Reply:
This was due to a git mistake.